### PR TITLE
E2E test coverage for BRC feature

### PR DESCRIPTION
**Pull request by WolfyUK**
### Context
[AB#222988](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/222988) [AB#235845](https://dfe-ssp.visualstudio.com/a14e55df-4fbf-4a2f-a11d-22b187178343/_workitems/edit/235845)

### Change proposed in this pull request
E2E tests against the `d02` environment.

### Guidance to review 
Run the tests targeting the above environment.

### Checklist (add/remove as appropriate)
- [x] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally
- [ ] ~~You have reviewed with UX/Design~~

### CODE REVIEW

File: web/tests/Web.E2ETests/Steps/School/HomeSteps.cs [hunk 1]
```diff
@@ -197,4 +197,11 @@ public async Task ThenTheRagGuidanceIsVisible()
         Assert.NotNull(_schoolHomePage);
         await _schoolHomePage.AssertRagGuidance();
     }
+
+    [Then("I should see a 404 error page indicating the BRC is not available for this school")]
+    public async Task xxx()
```
Comment (jrabbott) [inline web/tests/Web.E2ETests/Steps/School/HomeSteps.cs:202]: image

Comment (WolfyUK) [reply web/tests/Web.E2ETests/Steps/School/HomeSteps.cs]: 🫣

**Comment by jrabbott** [comment]
@Faizan-ahmad00 would you be able to review this PR?

**Review by Faizan-ahmad00 — APPROVED** [review]
looks good to me. there might be one more scenario for non-lead federation school but that will be after the specs have been finalised.

File: web/tests/Web.E2ETests/Steps/School/HomeSteps.cs (hunk 1 above)
Comment (Faizan-ahmad00) [reply web/tests/Web.E2ETests/Steps/School/HomeSteps.cs]: @jrabbott are you referring to the wording of the step?

Comment (jrabbott) [reply web/tests/Web.E2ETests/Steps/School/HomeSteps.cs]: It's been resolved. The method name was triple x.

**Pull request merged**